src/AppConstants.js

```diff
@@ -9,7 +9,8 @@ export const SID_SET = 'SID_SET';
 
 // Compliance App Constants
 export const COMPLIANCE_FETCH = 'COMPLIANCE_SUMMARY_FETCH';
-export const COMPLIANCE_FETCH_URL = `${BASE_URL}/compliance/profiles?search=has_test_results=true`;
+export const COMPLIANCE_SEARCH = '(has_policy_test_results = true AND external = false) OR (has_policy = false AND has_test_results = true)';
+export const COMPLIANCE_FETCH_URL = `${BASE_URL}/compliance/profiles?search=${COMPLIANCE_SEARCH}`;
```

Comment:
Data passed through the URI params should be encoded. See [`encodeURI()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI).

Reply:
Updated. Thanks!